server/db/init.js

```diff
@@ -1,32 +1,30 @@
 import knexModule from 'knex';
 import bookshelfModule from 'bookshelf';
 
-const knex = knexModule({
-  client: 'pg',
-  connection: process.env.PG_CONNECTION_STRING,
-  debug: true,
-});
+const initializeDB = function(config, module) {
+  module(config);
+}
 
-const bookshelf = bookshelfModule(knex);
+const bookshelf = function(bookshelfObj) {
```

Comment:
different name for this function as it causes confusion later in other files when we import bookshelf from here and the bookshelf module from node_modules/